nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/enrichers/AbstractJsonEnricher.java

```diff
@@ -56,8 +64,20 @@ public boolean accept(Class<?> clazz, Type genericType, MediaType mediatype) {
 
     @Override
     public void write(Enriched<EntityType> enrichable, JsonGenerator jg) {
-        try {
-            write(jg, enrichable.getEntity());
+        try (Writer writer = new StringWriter()) {
+            try (TokenBuffer tb = new TokenBuffer(MAPPER, false)) {
+                // Write to a temporary output
```

Comment:
Expand the comment to: `Write to a temporary output in case of exception during write()`